### PR TITLE
Add recipe for KronaTools 2.5

### PR DESCRIPTION
**Pull request by davebx**

### CODE REVIEW

File: packages/package_krona_2_5/tool_dependencies.xml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+                    <source_directory>scripts</source_directory>
+                    <destination_directory>$INSTALL_DIR/scripts</destination_directory>
+                </action>
+                <action type="make_directory">$INSTALL_DIR/bin</action>
+                <action type="shell_command">
+                    for i in ClassifyBLAST GetContigMagnitudes GetLibPath GetTaxIDFromGI \
+                            ImportBLAST ImportDiskUsage ImportEC ImportFCP ImportGalaxy \
+                            ImportMETAREP-BLAST ImportMETAREP-EC ImportMGRAST ImportPhymmBL \
+                            ImportRDP ImportRDPComparison ImportTaxonomy ImportText ImportXML ; do
+                        ln -s $INSTALL_DIR/scripts/$i.pl $INSTALL_DIR/bin/$i
```
Comment (nsoranzo) [inline packages/package_krona_2_5/tool_dependencies.xml]: `s|$INSTALL_DIR/bin/$i|$INSTALL_DIR/bin/kt$i|`

**Comment by nsoranzo** [comment]
For https://toolshed.g2.bx.psu.edu/view/crs4/taxonomy_krona_chart/bd565e1e3c5f I've used this tool_dependencies.xml file for Krona: https://bitbucket.org/crs4/orione-tools/src/HEAD/custom/taxonomy_krona_chart/tool_dependencies.xml?at=bb-orione-tools

As you can see the install is done by moving all the sources and then simply using the provided install script:
```xml
<action type="shell_command">cd $INSTALL_DIR &amp;&amp; ./install.pl --prefix=$INSTALL_DIR</action>
```

**Comment by davebx** [comment]
Yeah, I chose this approach to avoid moving the tarball files into $INSTALL_DIR, leaving unneeded files cluttering up the filesystem.

**Comment by nsoranzo** [comment]
:+1:

**Pull request merged**